test/e2e/persistent_volume/persistent_volume_test.go

```diff
@@ -133,15 +134,16 @@ var _ = ginkgo.Describe("PersistentVolume", func() {
 		Ω(err).NotTo(HaveOccurred())
 	})
 
+	// Create nfs server and related rbac
```

Comment:
Looks like it's only naming there, the create operation is still on L147. I think it's better move the comment back!

Reply:
Yeah, good catch!